packages/ui/src/components/ColumnManager.tsx

```diff
@@ -28,29 +24,17 @@ function ColumnMenuItem<RowDataModel>({
     (typeof column.columnDef.header === 'string' ? column.columnDef.header : column.id);
 
   return (
-    <Dropdown.Item
-      key={column.id}
-      as="div"
-      class="px-2 py-1 d-flex align-items-center justify-content-between"
-      onKeyDown={onClearElementFocus}
```

Comment:
I genuinely couldn't tell what this was meant to do. I removed it. @reteps let me know if it has a purpose, and if so, please add a documentation comment.